mssentinel.json

```diff
@@ -0,0 +1,1072 @@
+{
+    "appid": "e6434377-a3e4-4a5f-bfef-4f37e53e0676",
+    "name": "Sentinel",
+    "description": "This app provides integration with Microsoft Sentinel",
+    "type": "information",
+    "product_vendor": "Microsoft",
+    "logo": "logo_mssentinel.svg",
+    "logo_dark": "logo_mssentinel_dark.svg",
+    "product_name": "Sentinel",
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "Splunk Community",
+    "license": "Copyright (c) Splunk Community, 2022",
```

Comment:
Shouldn’t the license value be "Copyright (c) 2022 Splunk Inc."?

Reply:
Updated license value as per standard license string.